Countly.js

```diff
@@ -1604,19 +1594,21 @@ Countly.getRemoteConfigValueForKeyP = function (keyName) {
         return 'To be implemented';
     }
     const promise = CountlyReactNative.getRemoteConfigValueForKeyP(keyName);
-    return promise.then((value) => {
-        if (Platform.OS == 'android') {
-            try {
-                value = JSON.parse(value);
-            } catch (e) {
-                // Countly.logError('getRemoteConfigValueForKeyP', e.message);
-                // noop. value will remain string if not JSON parsable and returned as string
+    return promise
+        .then((value) => {
+            if (Platform.OS == 'android') {
```

Comment:
I think we should use the same approach of platform/OS checks for consistency.
i.e "/android/.exec(Platform.OS)"
There are two other places in this file which are using == to check OS on line 1574 and 1593

Reply:
great catch